openstates/tx/bills.py

```diff
@@ -202,54 +211,54 @@ def scrape_bill(self, session, history_url):
             introduced = False
```

Comment:
I'm mistaken. A bill can have an array of classifications, and this variable makes sure it only tacks the word "introduction" on when it should. But with it being set to `False` at the top of the for loop, can it ever be `True` when adding a second action? I would think it gets reset for each `action`. What am I missing?